pkg/service/presentation/storage/storage.go

```diff
@@ -12,17 +14,21 @@ import (
 type StoredDefinition struct {
 	ID                     string                          `json:"id"`
 	PresentationDefinition exchange.PresentationDefinition `json:"presentationDefinition"`
+	Author                 string                          `json:"issuerID"`
+	AuthorKID              string                          `json:"issuerKid"`
 }
 
 type Storage interface {
 	DefinitionStorage
+	RequestStorage
 	SubmissionStorage
 }
 
 type DefinitionStorage interface {
-	StoreDefinition(schema StoredDefinition) error
-	GetDefinition(id string) (*StoredDefinition, error)
-	DeleteDefinition(id string) error
+	StoreDefinition(ctx context.Context, presentation StoredDefinition) error
+	GetDefinition(ctx context.Context, id string) (*StoredDefinition, error)
+	DeleteDefinition(ctx context.Context, id string) error
+	ListDefinitions(ctx context.Context) ([]StoredDefinition, error)
```

Comment:
our other APIs use `GetDefinitions` instead of `List` - I'm ok with list but let's keep them all in line

Reply:
Created https://github.com/TBD54566975/ssi-service/issues/449